src/libOpenImageIO/maketexture.cpp

```diff
@@ -758,7 +758,9 @@ write_mipmap(ImageBufAlgo::MakeTextureMode mode, std::shared_ptr<ImageBuf>& img,
                 mipimages.erase(mipimages.begin());
             } else {
                 // Resize a factor of two smaller
-                smallspec        = outspec;
+                smallspec = outspec;
+                if (!configspec.get_int_attribute("maketx:allow_pixel_shift"))
```

Comment:
Looks like a copy/paste typo?
```suggestion
                if (!configspec.get_int_attribute("maketx:mipmap_metadata"))
```

Reply:
I'm not quite sure how I did that, sure does look like c/p problem. Thanks for spotting it.